src/migration/1678256072490-PostOptionality.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class PostOptionality1678256072490 implements MigrationInterface {
+  name = 'PostOptionality1678256072490';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "post" DROP COLUMN "tweeted"`);
+    await queryRunner.query(
+      `ALTER TABLE "post" ALTER COLUMN "title" DROP NOT NULL`,
+    );
+    await queryRunner.query(
+      `INSERT INTO "public"."source" ("id", "name", "handle", "private") VALUES ('unknown', 'unknown', 'unknown', 'true') ON CONFLICT DO NOTHING`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "post" ALTER COLUMN "sourceId" SET DEFAULT 'unknown'`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "post" ALTER COLUMN "sourceId" SET NOT NULL`,
+    );
```

Comment:
I think we already have a migration for this: https://github.com/dailydotdev/daily-api/blob/main/1677643070022-UnknownSource.ts

Reply:
Ah I thought so, but it won't run indeed from main so I didn't see it reflect in my DB

Reply:
Right. Feel free to delete the one in the root folder then I will close the PR itself.

Reply:
Done 🙏